woocommerce_fusion/woocommerce/doctype/woocommerce_server/woocommerce_server.py

```
@@ -142,10 +237,69 @@ def get_item_docfields(self, doctype: str) -> List[dict]:
 		"""
-		return [key for key in WC_ORDER_STATUS_MAPPING.keys()]
+		allowed = self.get("allowed_inbound_statuses")
+		parsed: List[str] | None = None
+		if isinstance(allowed, list):
+			parsed = allowed
+		elif isinstance(allowed, str) and allowed.strip():
+			try:
+				data = frappe.parse_json(allowed)
+				if isinstance(data, list):
+					parsed = data
+			except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)